jenkins-jobs/git/git-notify-ib-updates

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+
+import datetime
+import json
+import logging
+import sys
+import os
+import urllib.error
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "shift"))
+try:
+    from github_utils import github_api
+    import libib
+
+    # noinspection PyUnresolvedReferences
+    from libib import PackageInfo, ErrorInfo
+except ImportError:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "shift"))
+    from github_utils import github_api
+    import libib  # NOQA
+
+    # noinspection PyUnresolvedReferences
+    from libib import PackageInfo, ErrorInfo
```

Comment:
@iarspider , I do not under the logic here. are then two `sys.path` at line number 10 and 11 enough? why you have to insert the same path again?

Reply:
@smuzaffar these lines were for local testing. Cleaned up.